### ISSUE TITLE
Pytorch thingys

### ISSUE THREAD

**Issue by YugeTen** [YugeTen/papers-log#27]
[create own validation set in dataloader](https://gist.github.com/kevinzakka/d33bf8d6c7f06a9d8c76d97a7879f5cb)

**Comment by YugeTen** [comment]
# HowTos
1. [Conda VM, PyTorch 1.0 installation and set up](https://gist.github.com/YugeTen/30aef78eabc800398ca12cf3b225c8f0)

**Comment by YugeTen** [comment]
# Python
1. [basic] (https://gist.github.com/YugeTen/25883ae8be880a9894d9137afce8251e)

**Comment by YugeTen** [comment]
# Git
1. [Git hacks: branching, merging, history and more](https://gist.github.com/YugeTen/f182e86a8d7b68a72c5220ecc9ac6ea0) ([ref](https://learngitbranching.js.org/))

**Comment by YugeTen** [comment]
# Write a good paper[https://gist.github.com/YugeTen/4d3ba1e75b3f1e85c9c68b79b3515baf]
